src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
```

Comment:
<img width="367" alt="Screenshot 2023-11-03 at 14 49 12" src="https://github.com/mate-academy/layout_calendar/assets/135232001/36eb420a-ba71-42c7-871f-b1b1ffd7b84c">
You can add these styles to center your calendar on the page